app/src/main/java/com/beemdevelopment/aegis/ui/views/EntryListView.java

```diff
@@ -144,6 +146,10 @@ public int getSpanSize(int position) {
             @Override
             public void onRefresh() {
                 refresh(false);
+
+                if (_recyclerView.isShown()) {
+                    VibrationHelper.vibratePattern(getContext(), VibrationPatterns.REFRESH_CODE);
```

Comment:
I'm not sure if this is only the case on my device, but I don't feel this vibration.